### PR TITLE
enhancement: detect sensitives in memo, IP:SR

### PR DESCRIPTION
**Pull request by theghost1980**

### CODE REVIEW

File: src/common-ui/confirmation-page/confirmation-page.component.tsx [hunk 1]
```diff
@@ -98,6 +106,8 @@ const mapStateToProps = (state: RootState) => {
     message: state.navigation.stack[0].params.message as string,
     fields: state.navigation.stack[0].params.fields as ConfirmationPageFields[],
     warningMessage: state.navigation.stack[0].params.warningMessage as string,
+    privateKeyMemoValidationWarning: state.navigation.stack[0].params
```
Comment (tshiuan) [inline src/common-ui/confirmation-page/confirmation-page.component.tsx:109]: you already have a warning message field. Not need to create a new one :)

Comment (cedricguillas) [inline src/common-ui/confirmation-page/confirmation-page.component.tsx:109]: no need to create a new field, you have a warningMessage field right above

Comment (theghost1980) [reply src/common-ui/confirmation-page/confirmation-page.component.tsx:109]: Ok, thanks.

**Comment by stoodkev** [comment]
Hi @theghost1980 , is this ready for review? If so, please remove the draft status of the keychain-commons branch and let me know. Thanks

**Pull request merged**